src/Microsoft.OpenApi/Attributes/DisplayAttribute.cs

```diff
@@ -15,14 +15,14 @@ public class DisplayAttribute : Attribute
         /// Initializes a new instance of the <see cref="DisplayAttribute"/> class.
         /// </summary>
         /// <param name="name">The display name.</param>
-        public DisplayAttribute(string name)
+        public DisplayAttribute(string? name)
         {
             Name = Utils.CheckArgumentNullOrEmpty(name);
         }
 
         /// <summary>
         /// The display Name.
         /// </summary>
-        public string Name { get; }
+        public string? Name { get; }
```

Comment:
revert this, it's assigned in the only constructor and checked it's not null. I suspect the Utils method return type needs to be updated instead.